.github/workflows/tester.yml

```diff
@@ -34,6 +34,8 @@ jobs:
           github-token: ${{ secrets.github_token }}
           flag-name: run-${{ runner.os }}-${{ matrix.node-version }}
           parallel: true
+      - name: Benchmark
+        run: node script/benchmark.js
```

Comment:
Currently we run benchmark on Travis CI: https://github.com/hexojs/hexo/blob/master/.travis.yml

Would you mind adding the benchmark script in `.travis.yml` instead?